lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -5,3 +5,38 @@
  */
 
 include_once 'dosomething_global.features.inc';
+define('GLOBAL_ADMIN_ROLES', 'mexio admin,brazil admin');
+
+/**
+ * Implements hook_form_alter().
+ */
+function dosomething_global_form_campaign_node_form_alter(&$form, &$form_state, $form_id) {
+  // Check if the user is a mexico or brazil admin
+  if (dosomething_global_is_regional_admin()) {
+    // Get translatable fields.
+    $fields = $form_state['field'];
+    foreach ($fields as $field_name => $field) {
+      $field_info = field_info_field($field_name);
+      // If the field is not translatable disable access
+      if (!$field_info['translatable']) {
+        $form[$field_name]['#access'] = FALSE;
+      }
+    }
+  }
+}
+
+/**
+ * Returns if the user is a regional admin
+ *
+ * @param Object $user
+ *   Optional - A specified user to check
+ *
+ * @return SelectQuery object
+ */
+function dosomething_global_is_regional_admin($user = NULL) {
+  if (!isset($user)) {
+    global $user;
+  }
+  $roles = explode(",", GLOBAL_ADMIN_ROLES);
+  return in_array('brazil admin', $roles);
```

Comment:
this will always return true if you are just testing if brazil admin is in the `global_roles` array